tests/dmctl_basic/check_list/check_task.sh

```diff
@@ -34,3 +34,33 @@ function check_task_error_database_config() {
         "Access denied for user" 1 \
         "Please check the database connection and the database config in configuration file" 1
 }
+
+function check_task_error_count() {
+    # 10 errors
+    run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "check-task $cur/conf/dm-task3.yaml" \
+        "\"result\": false" 1 \
+        "\"failed\": 2" 1 \
+        "\"state\": \"fail\"" 2
+
+    # 1 error
+    run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "check-task $cur/conf/dm-task3.yaml -e 1" \
+        "\"result\": false" 1 \
+        "\"failed\": 2" 1 \
+        "\"state\": \"fail\"" 1
+
+    # 100 errors
+    run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "check-task $cur/conf/dm-task3.yaml -e 100 -w 1" \
```

Comment:
Use `--error` and `--warn` here?
```suggestion
        "check-task $cur/conf/dm-task3.yaml --error 100 --warn 1" \
```